src/main/java/de/cyface/collector/handler/MeasurementHandler.java

```diff
@@ -99,18 +99,18 @@ public void handle(final RoutingContext ctx) {
                 response.end();
             }
         } catch (final NumberFormatException e) {
-            LOGGER.error("Data was not parseable!");
+            LOGGER.error("Data was not parsable!");
             ctx.fail(422);
         }
 
     }
 
     /**
      * Informs the system about a new measurement that has arrived.
-     * 
+     *
      * @param measurement The newly arrived measurement.
-     * @param context The routing context necessary to get access to the Vert.x
-     *            event bus.
+     * @param context     The routing context necessary to get access to the Vert.x
```

Comment:
This is not how our formatter should do it? Is this some IntelliJ specific setting?